originals/a.yaml

```
@@ -214,6 +214,22 @@
     - Horror
     - Sci-Fi
 
+- name: Aliens versus Predator Classic 2000
+  names:
+  - Aliens versus Predator Gold
+  - Aliens versus Predator 1999
```

```suggestion
- name: Aliens Versus Predator
```
the other names are very similar so not really needed